### DISCUSSION TITLE
Slash Commands

### DISCUSSION THREAD

**Discussion by brahmendra79** [SudhanPlayz/Discord-MusicBot#648] [Q&A] [answered]
Some  Slash Commands Are Not Laoding

**Comment by ItzSudhan** [comment, marked as answer]
Re-deploy the slash commands again, Run `yarn deploy`

**Reply by brahmendra79** [reply]
How To Re Deploy

**Reply by brahmendra79** [reply]
I Am Using Replit
 How TO Deploy Please Say

**Reply by brahmendra79** [reply]
Evey Thing Loaded But Lavalink Is Not Connecting I Have Tried That Link In Previous Version Of Your Bot Its Working Fine But When It Comes To This V5 Bot Its Not Connecting To Lavalink

**Reply by brahmendra79** [reply]
I Am Using Lavalink In Repl.it Only

**Reply by ItzSudhan** [reply]
> Evey Thing Loaded But Lavalink Is Not Connecting I Have Tried That Link In Previous Version Of Your Bot Its Working Fine But When It Comes To This V5 Bot Its Not Connecting To Lavalink

Fix your lavalink server or switch to working lavalink

**Reply by ItzSudhan** [reply]
> How To Re Deploy

Open shell and run `yarn deploy`

**Reply by brahmendra79** [reply]
error Command "deploy" not found.
